### PR TITLE
Added tooltip when hovering over No-Data

### PR DESCRIPTION
**Pull request by gantanikhilraj**
Closes issue #3557 
When hovering over the "No Data" it will show  "Unscheduled events will show up here" or "x events are unscheduled"  based on the present data.

#### PR Checklist

- [ ] My code adheres to [AppFlowy's Conventions](https://docs.appflowy.io/docs/documentation/software-contributions/conventions)
- [ ] I've listed at least one issue that this PR fixes in the description above.
- [ ] I've added a test(s) to validate changes in this PR, or this PR only contains semantic changes.
- [ ] All existing tests are passing.

### CODE REVIEW

File: frontend/appflowy_flutter/lib/plugins/database_view/calendar/presentation/calendar_page.dart [hunk 1]
```diff
@@ -375,9 +375,14 @@ class _UnscheduledEventsButtonState extends State<UnscheduledEventsButton> {
                   _popoverController.show();
                 }
               },
-              child: FlowyText.regular(
-                "${LocaleKeys.calendar_settings_noDateTitle.tr()} (${state.unscheduleEvents.length})",
-                fontSize: 10,
+              child: Tooltip(
+                message: state.unscheduleEvents.isEmpty
+                    ? "Unscheduled events will show up here"
```
Comment (MayurSMahajan) [inline frontend/appflowy_flutter/lib/plugins/database_view/calendar/presentation/calendar_page.dart:380]: Localize this message

File: frontend/appflowy_flutter/lib/plugins/database_view/calendar/presentation/calendar_page.dart [hunk 2]
```diff
@@ -375,9 +375,14 @@ class _UnscheduledEventsButtonState extends State<UnscheduledEventsButton> {
...
+                    : "${state.unscheduleEvents.length} events are unscheduled",
```
Comment (MayurSMahajan) [inline frontend/appflowy_flutter/lib/plugins/database_view/calendar/presentation/calendar_page.dart:381]: Localize this message with a placeholder

**Review by MayurSMahajan — CHANGES_REQUESTED** [review]

File: frontend/appflowy_flutter/lib/plugins/database_view/calendar/presentation/calendar_page.dart (hunk 2 above)
Comment (gantanikhilraj) [reply frontend/appflowy_flutter/lib/plugins/database_view/calendar/presentation/calendar_page.dart:381]: Can you provide me some examples, to understand this more clearly.

Comment (0xJaskeerat) [reply frontend/appflowy_flutter/lib/plugins/database_view/calendar/presentation/calendar_page.dart:381]: Am guessing here you mean , storing the string in the common .json file and creating a LocaleKeys to get it @MayurSMahajan

Comment (LucasXu0) [reply frontend/appflowy_flutter/lib/plugins/database_view/calendar/presentation/calendar_page.dart:381]: - Add the translations to AppFlowy/frontend/resources/translations/en.json.
- Execute the command sh scripts/code_generation/language_files/generate_language_files.sh in the frontend directory.
- Use something similar to LocaleKeys.document_imageBlock_upload_placeholder.tr() to replace the hardcoded text.

**Comment by richardshiue** [comment]
Hi @gantanikhilraj, how much progress have you made on this? Do you need any help?

**Pull request merged**